src/Endpoints/WorkersKV.php

```diff
@@ -0,0 +1,97 @@
+
+    private $adapter;
+
+    public function __construct(Adapter $adapter)
+    {
+        $this->adapter = $adapter;
+    }
+
+    public function createNamespace(string $accountID, string $namespace): \stdClass
+    {
+        $response = $this->adapter->post(
+            'accounts/' . $accountID . '/storage/kv/namespaces',
+            ['title' => $namespace]
+        );
+        $this->body = json_decode($response->getBody());
+        return $this->body->result;
+    }
+
+    public function getNameSpaces(string $accountID): array
+    {
+        $response = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces');
+        $this->body = json_decode($response->getBody());
+        return $this->body->result;
+    }
+
+    public function listNamespaceKeys(string $accountID, $namespaceIdentifier): array
+    {
+        $response = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/keys');
+        $this->body = json_decode($response->getBody());
+        return $this->body->result;
+    }
+
+    public function getReadKeyValuePair(string $accountID, $namespaceIdentifier, $keyName): string
+    {
+        $response = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/values/' . $keyName);
+        $this->body = json_decode($response->getBody());
+        return $this->body;
+    }
+
+    public function getAllKeysAndValuesForNamespace(string $accountID, string $namespaceIdentifier)
+    {
+        $keys = $this->listNamespaceKeys($accountID, $namespaceIdentifier);
+        foreach ($keys as $index => $values) {
+            $value = $this->getReadKeyValuePair($accountID, $namespaceIdentifier, $values->name);
+            $keys[$index]->value = $value;
+        }
+        return $keys;
+    }
+
+    public function getListOfNamespaces(string $accountID, int $page = 1, int $perPage = 50, string $order = '', string $direction = '')
+    {
+        $query = [
+            'page' => $page,
+            'per_page' => $perPage
+        ];
+
+        if (!empty($order)) {
+            $query['order'] = $order;
+        }
+
+        if (!empty($direction)) {
+            $query['direction'] = $direction;
+        }
+
+        $namespace = $this->adapter->get('accounts/' . $accountID . '/storage/kv/namespaces', $query);
+        $this->body = json_decode($namespace->getBody());
+        return $this->body->result;
+    }
+
+    public function writeMultipleKeyValuePairs(string $accountID, $namespaceIdentifier, $keys = []): bool
+    {
+        $this->adapter->put('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/bulk', $keys);
+        return true;
+    }
+
+    public function deleteKeyValuePair(string $accountID, $namespaceIdentifier, string $key): bool
+    {
+        $this->adapter->delete('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/values', [$key]);
```

Comment:
I've just had to use this method and it looks like this might be an old way of deleting the KV pair as it is now throwing a 404 error.
According to the documentation the key should be part of the url instead of the body:
https://api.cloudflare.com/#workers-kv-namespace-delete-key-value-pair

```suggestion
        $this->adapter->delete('accounts/' . $accountID . '/storage/kv/namespaces/' . $namespaceIdentifier . '/values/' . $key);
```